ci-operator/step-registry/storage/destroy/must-gather-before-destroy/storage-destroy-must-gather-before-destroy-commands.sh

```diff
@@ -0,0 +1,287 @@
+#!/bin/bash
```

Comment:
Have you considered a symlink? If so, add a comment here that this is a copy for <xyz> and symlink does not work.

Reply:
I have tried symlink and it doesn't work